### PR TITLE
Upstream Merge Oct 29 2024

### PR DESCRIPTION
**Pull request by zelezniciar1**
# Much description, very descript

Description.

---

# TODO

- [ ] Task
- [x] Completed Task

---

<details><summary><h1>Media</h1></summary>
<p>

Example Media Embed

</p>
</details>

---

# Changelog

### CODE REVIEW

**Review by SsalamethVersaach — APPROVED** [review]

File: Resources/Prototypes/GameRules/roundstart.yml
Comment (zelezniciar1) [inline Resources/Prototypes/GameRules/roundstart.yml:1]: Fixed the merge conflict introduced in roundstart.yml by #124

**Review by SsalamethVersaach — APPROVED** [review]

**Comment by sleepyyapril** [comment]
Is this fine to merge? Comment appears to not have been been committed?

**Comment by SsalamethVersaach** [comment]
> Is this fine to merge? Comment appears to not have been been committed?

Yeah, TManz just needs to review it because I'm not force merging stuff anymore, just to make sure things work for branch protection.

**Comment by Tmanzxd** [comment]
Yup reviewing it now.

**Review by Tmanzxd — APPROVED** [review]
Builds and runs with no issues. Be sure to watch for bugs during gameplay.

**Comment by Tmanzxd** [comment]
Also @sleepyyapril be sure to synchronize changes before building after merging.

**Pull request merged**